elkm1_lib/keypads.py

```diff
@@ -20,7 +23,13 @@ def __init__(self, index: int, connection: Connection, notifier: Notifier) -> No
         self.last_user = -1
         self.code = ""
         self.last_keypress: Optional[tuple[str, int]] = None
+        # Not sure what these init values should be? should it be same as above?
```

Comment:
The values you have are good. The general pattern is `None` unless there's something that will cause more problems.

Reply:
Using None is ok, just requires checking for that when referencing it, since once it's defined we get an array.  I'll remove the comment.